<!DOCTYPE html>
<html>
  <head>
    <meta name="pypi:repository-version" content="1.1">
    <title>Links for pys-django-issue-tracker</title>
  </head>
  <body>
    <h1>Links for pys-django-issue-tracker</h1>
    <a href="https://files.pythonhosted.org/packages/ac/47/d36ec4b92e0044953ba3d6b5da576affa481ba83fc3ce02e3ed8ed471fd4/pys-django-issue-tracker-1.0.0.tar.gz#sha256=2c1c03e962be027e61da45d3cc16b316992dfb68d99f54586b0f0e34bc123088" data-requires-python="&gt;=3.5" >pys-django-issue-tracker-1.0.0.tar.gz</a><br />
    <a href="https://files.pythonhosted.org/packages/b3/03/3361ec19fcf4a64912bf9d31a0824f7df843244029c33075a9272beb784a/pys_django_issue_tracker-1.0.0-py3-none-any.whl#sha256=25501f2a12a9047b5101c774c966c505b89eee7eb538e22a6bea5b840223ad0c" data-requires-python="&gt;=3.5" data-dist-info-metadata="sha256=c14b038b74676cd261253cd640460435d767c7278280077de1fe1f8a0a03a596">pys_django_issue_tracker-1.0.0-py3-none-any.whl</a><br />
    <a href="https://files.pythonhosted.org/packages/7e/bf/d86dcac4be47f4fad7f67814edec365a9d4963f0b1da944a7d4c52628409/pys-django-issue-tracker-1.0.1.tar.gz#sha256=d03ab840cc39ae364873b67177bee4511ba725ed712853b50d2ba1e48577a6a4" data-requires-python="&gt;=3.5" >pys-django-issue-tracker-1.0.1.tar.gz</a><br />
    <a href="https://files.pythonhosted.org/packages/4a/48/375b529f14e8d489f85d2bf5faad1409734fd908402e73526285594de793/pys_django_issue_tracker-1.0.1-py3-none-any.whl#sha256=46c84a4e58492eeb6fded68b51aec77ae21c80c96c0624077f37e196f8ba3caa" data-requires-python="&gt;=3.5" data-dist-info-metadata="sha256=1f41cc9de330de67524c2c337dd3a2a2e9b837a486ade92432d63ba812cb96da">pys_django_issue_tracker-1.0.1-py3-none-any.whl</a><br />
    </body>
</html>
<!--SERIAL 18511570-->